homeassistant/components/advantage_air/entity.py

```diff
@@ -18,11 +18,11 @@ def __init__(self, instance, ac_key, zone_key=None):
         self.ac_key = ac_key
         self.zone_key = zone_key
         self._attr_device_info = DeviceInfo(
-            identifiers={(DOMAIN, self.coordinator.data["system"]["rid"])},
+            via_device=(DOMAIN, self.coordinator.data["system"]["rid"]),
+            identifiers={(DOMAIN, self.coordinator.data["system"]["rid"], ac_key)},
```

Comment:
The identifier should be a two string tuple. Please join the ac_key string with the system rid string. Normally we separate identifier strings with underscore or similar.

https://github.com/home-assistant/core/blob/11f80762eabf9e0a3c21395e7ae9cd8d939016ea/homeassistant/helpers/device_registry.py#L85

Reply:
Done, thanks @MartinHjelmare